src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-upgrade-notes-lts-2021.md

```diff
@@ -82,6 +82,28 @@ From now on, retrieving tasks needs to be made using a paginable approach. Some
 
 As of [NXP-28263](https://jira.nuxeo.com/browse/NXP-28263), `nuxeo-dropzone` exposes a new API that allows the element to be bound to a document field through the *value property*, which is more consistent with the API exposed by the other widgets. The previous API is deprecated but still supported (see [NXP-29391](https://jira.nuxeo.com/browse/NXP-29391)).
 
+### Routing Behavior API Change
+
+In order to support navigation by document UID ([WEBUI-14](https://jira.nuxeo.com/browse/WEBUI-14)) and cross repository navigation ([ELEMENTS-1211](https://jira.nuxeo.com/browse/ELEMENTS-1211)), the API of the Web UI Routing Behavior was extended. Previously, methods like `urlFor()` and `navigateTo()` would make use of two arguments:
+```
+// generates the URL to browse to the doc document
+urlFor('browse', doc.path)
+
+// navigates the UI to the browse view of the doc document
+navigateTo('browse', doc.path)
+```
+Currently, those same methods can also be used by passing the document object as the only argument:
```

Comment:
We can leave a note that this is the recommended way of doing it unless you explicitly need to specify the route.